thoth/storages/graph/postgres.py

```diff
@@ -4679,6 +4679,53 @@ def sync_dependency_monkey_result(self, document: dict) -> None:
 
                 inspection_run.dependency_monkey_run_id = dependency_monkey_run.id
 
+    def get_python_package_required_symbols(
+        self, package_name: str, package_version: str, index_url: str
+    ) -> Optional[List[str]]:
+        """Get required symbols for a Python package in a specified version."""
+        package_name = self.normalize_python_package_name(package_name)
+        package_version = self.normalize_python_package_version(package_version)
+
+        with self._session_scope() as session:
+            query = session.query(PythonPackageVersion)
+
+            query = (
+                query.filter(PythonPackageVersion.package_name == package_name)
+                .filter(PythonPackageVersion.package_version == package_version)
+                .join((HasArtifact, PythonPackageVersionEntity.python_artifacts))
+                .join((PythonArtifact, HasArtifact.python_artifact))
```

Comment:
I think you can use `join` without explicitly stating relation - sqlalchemy will use the default one:

```python
                .join(HasArtifact, PythonPackageVersionEntity)
                .join(PythonArtifact)
```